src/basics/basics.md

```diff
@@ -1,8 +1,6 @@
 # Basics
 
-Now that we've successfully compiled a program and executed it within the context of our Arch VM, let's dig into the details of how an Arch [Program] is written and discuss some best practices.
-
-In this section, we'll guide you through the basics of writing functions and handling input parameters effectively within a demo program: [GraffitiWall].
+To illustrate the basic process involved in developing an Arch [program], we're going to build, deploy and interact with our demo app: [GraffitiWall].
```

Comment:
Minor detail but this will not render the link correctly since Program is capitalized and the reference is lowercased within the sentence.

Reply:
https://github.com/Perelyn-Arch/arch-book/commit/7f68f33606173030f4be5fddd42b2850cd288eef